rui-select/template.hbs

```diff
@@ -0,0 +1,14 @@
+<select {{action 'change' on='change'}} class='form-control'>
+  {{#if prompt}}
+    <option disabled selected={{is-not selection}}>
+      {{prompt}}
+    </option>
+  {{/if}}
+
+  {{#each content key="@identity" as |item|}}
+    <option value="{{read-path item optionValuePath}}" selected={{is-equal item.id selectedOption.id}}>
+      {{read-path item optionLabelPath}}
+     </option>
+
+  {{/each}}
+</select>
```

Comment:
Not sure how these got in here.. I think I was doing some work on my work repository that uses these components and did some bad copy pasta.